### ISSUE TITLE
Vignettes and documentation

### ISSUE THREAD

**Issue by lmsimp** [ComputationalProteomicsUnit/pRolocGUI#27]
## A new getting started with `pRolocGUI` vignette
- [x] What is `pRolocGUI` used for
- [x] Update screen shots of Apps
- [x] Walk through examples for each app

## Other
- [x] Documentation - one .Rd file for all Apps
- [x] Talk to @lgatto about new videos for apps once above done - see issue #33

**Comment by lmsimp** [comment]
@lgatto Should I do this in the Apps repo or start it here under a new branch?

**Comment by lmsimp** [comment]
Will need to update/replace [this](http://computationalproteomicsunit.github.io/pRolocGUI/)

**Comment by lgatto** [comment]
My plan was to automatically build to vignette as the README file.

**Comment by lmsimp** [comment]
Assumed we would, was more of a note to myself

**Comment by lmsimp** [comment]
I am re-writing the vignette at the moment. What to do about `plotMat2D`?

**Comment by lgatto** [comment]
Is `plotMat2D`s functionality not addressed in the *official* apps? I would keep it for now, but not export it yet. Let's see what is missing once the overview vignette is written.

**Comment by lmsimp** [comment]
Is there any reason why the current `pRolocGUI` vignette is html and the vignettes for `pRoloc` and `MSnbase` are not?

**Comment by lgatto** [comment]
Mainly historical. But I prefer to keep it html, as it can be more easily read online. Also, I would prefer if you used [BiocStyle](http://www.bioconductor.org/packages/release/bioc/html/BiocStyle.html).

**Comment by lmsimp** [comment]
@lgatto **Update:** 

* The vignette is now up-to-date but explaining the compare application is tricky as it's quite a complicated application and there is already documentation.  
* What should we do about compare app? I would like to re-write it so it's pretty much the same as the PCA app but with two plots. I can open a new issue for this and we can discuss?   
* With regard to writing the paper, I think we first need to show some nice use cases e.g. hyperLOPIT data, for the profiles app would be nice to show how to discriminate between clusters and peroxisome and mitochondria would be good for this as they are overlapping in PCA etc.

**Comment by lgatto** [comment]
> At the moment I refer to this other vignette, but this old vignette needs updating if we refer to it. 

Yes, that's fine. I would like to get all the other apps/docs ready before removing anything. 

> I think the best option is to re-write the compare app it so it's pretty much the same as the PCA app but with two plots. I can open a new issue for this and we can discuss?

Ok for issue, but let's make sure all other apps are 100% before starting this new one.

Ack your points re application note. Not sure why you would want to turn the animations into apps?

**Comment by lmsimp** [comment]
Add `plot2Ds` and `move2Ds` on the end of the vignette.

As far as vis in `pRoloc` goes we have:

* plot2D
* plotDist
* plot2Ds
* move2Ds 

Can you add to the list @lgatto want to re-cap on Monday what we have already in `pRoloc`

**Comment by lgatto** [comment]
I think the list is complete. What I really wanted to emphasise in this visualisation vignette were basic visualisation principles such as what are PCA or profile plots good for (this is now explained in the GUI vignette) and describe the different dimensionality reduction methods available. The latter could probably go in the plot2D documentation at this stage. 

The last two functions are not described anywhere, so it might be good to briefly mention them after the `compare` app. I don't think we should go into too many details at this stage.

**Comment by lmsimp** [comment]
`move2Ds` does not work in RStudio but seems to work fine when using R from the command line where XQuartz is launched for graphics. This of course does not matter for today, but is something we should fix.

Also, where and what is the name of the function that you have written to show all dim reduction plots at once?

**Comment by lgatto** [comment]
> `move2Ds` does not work in RStudio but seems to work fine when using R from the command line where XQuartz is launched for graphics. This of course does not matter for today, but is something we should fix.

Could you open an issue in `pRoloc` for this?

> Also, where and what is the name of the function that you have written to show all dim reduction plots at once?

See `SpatProtVis`.
